lld/COFF/Config.h

```diff
@@ -115,6 +115,7 @@ struct Configuration {
   enum ManifestKind { Default, SideBySide, Embed, No };
   bool is64() const { return llvm::COFF::is64Bit(machine); }
 
+  llvm::SmallVector<uint8_t> stub;
```

Comment:
We could use `std::unique_ptr<MemoryBuffer>` here to avoid copying the blob in `parseStub`?

Reply:
Yes :).